plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -81,6 +85,19 @@ public PinotConfig setControllerUrls(String controllerUrl)
         return this;
     }
 
+    public Optional<HostAndPort> getBrokerUrl()
+    {
+        return brokerUrl;
+    }
+
+    @Config("pinot.broker-url")
```

Comment:
(No requested change) This is not "URL" to be exact. It's good for consistency with controller property though.